src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7893,7 +7860,7 @@ or SyntaxKind.MinusMinusToken
             isCollectionExpression = isCollectionExpression
                 || IsExpectedBinaryOperator(this.CurrentToken.Kind)
                 || IsExpectedAssignmentOperator(this.CurrentToken.Kind)
-                || this.CurrentToken.Kind is SyntaxKind.DotDotToken
+                || IsAtDotDotToken()
```

Comment:
I couldn't find a test affected by this line (I removed it and ran the tests from four test files in this PR). Could we add one?

Reply:
I'm still unable to find an affected test